app/build.gradle

```diff
@@ -9,7 +9,7 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'androidx.navigation.safeargs'
 
 def STRIPE_API_TOKEN = System.getenv('STRIPE_API_TOKEN') ?: "YOUR_API_KEY"
-def MAPBOX_KEY = '"pk.eyJ1IjoiYW5nbWFzMSIsImEiOiJjanNqZDd0N2YxN2Q5NDNuNTBiaGt6eHZqIn0.BCrxjW6rP_OuOuGtbhVEQg"'
+def MAPBOX_KEY = '"'+System.getenv('MAPBOX_KEY')+'"' ?: '"pk.eyJ1IjoiYW5nbWFzMSIsImEiOiJjanNqZDd0N2YxN2Q5NDNuNTBiaGt6eHZqIn0.BCrxjW6rP_OuOuGtbhVEQg"'
```

Comment:
Did as you said @iamareebjamal now I'm getting null as value of key. Sorry, can you tell me how to initialize the value of environment variable. I don't know how to do that.

Reply:
It shouldn't be null. If it is not defined in the environment, it should fallback to this key

Reply:
Even I thought so but it's not working

Reply:
Remove `'"'+` before and after

Reply:
resolved